spoon-runner/src/main/java/com/squareup/spoon/html/HtmlUtils.java

```diff
@@ -191,7 +192,9 @@ static ExceptionInfo processStackTrace(StackTrace exception) {
     if (exception == null) {
       return null;
     }
-    String message = exception.toString().replace("\n", "<br/>");
+    // Escape any special HTML characters in the exception that would otherwise break the HTML
+    // rendering (e.g. the angle brackets around the default toString() for enums).
+    String message = StringEscapeUtils.escapeHtml4(exception.toString());
```

Comment:
I drafted a change to do that too here: https://github.com/daj/spoon/commit/de202062ceaa27b1396aebd3a91faf1b9e85954d  Let me know if that looks okay and I'll add it to this PR.  

Also, to confirm - are you definitely happy that I've removed the `replace("\n", "<br/>")`?  Even though I didn't seem to need it, I wasn't sure if there are some scenarios where "\n" still need to be removed.  I can leave it in if you are worried.